src/Tools/Automation/ActionRunner/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+
+namespace ActionRunner
+{
+	internal class Program
+    {
+		private const string UnoPathPrefix = "uno/src/Uno.UI/Microsoft/UI/Xaml/Controls";
+		private const string MUXPathPrefix = "microsoft-ui-xaml/dev";
+
+		private static int Main()
+        {
+			if (!Directory.Exists(MUXPathPrefix))
+			{
+				GitHubLogger.LogError($"Can't find '{MUXPathPrefix}' directory.");
+				return 1;
+			}
+			else if (!Directory.Exists(UnoPathPrefix))
+			{
+				GitHubLogger.LogError($"Can't find '{UnoPathPrefix}' directory.");
+			}
+
+			string[] unoResourceFiles = CollectResourcesFromUno();
+			GitHubLogger.LogInformation($"Found {unoResourceFiles.Length} resources files.");
+
+			foreach (string unoPath in unoResourceFiles)
```

Comment:
Note: the workflow currently will only update existing files, but won't create new ones. Probably it should also create new ones as well?

Reply:
For now copying XAML styles is tricky, as many of them need Uno specific adjustments and we also are in process of adding support for WinUI 2.6. I would suggest postponing this after we figure out what will be the structuring/how to deal with the specifics

Reply:
Regarding new resources - we could probably include them, as it will not impact the assembly size at all, just it could be a bit confusing, that the folders for the controls are there while the controls are not. What do you think?

Reply:
> For now copying XAML styles is tricky, as many of them need Uno specific adjustments and we also are in process of adding support for WinUI 2.6. I would suggest postponing this after we figure out what will be the structuring/how to deal with the specifics

What I was planning to do here isn't really a "copy/paste". But get the diff and try to apply it. If there are conflicts, the bot will open an issue, otherwise, it will open a PR. I'm not sure if that would be easy enough, but worth trying. Let's look into that after adding support for WinUI 2.6.

For the new resources, I won't include ones from controls that don't exist. Only for existing controls (ie, if a new language was added in WinUI repo for an existing control). Let me know if you prefer that to be in a separate PR or this one.